README.md

```diff
@@ -148,6 +148,7 @@ const password = 'test'
 ssh.connect({
   host: 'localhost',
   username: 'steel',
+  port: '2222',
```

Comment:
Could you please change this to the default ssh port to make this copy paste friendly, also while a string works for port, could you please change this to a number

Reply:
Thank you for the review. I fixed it.